src/plone/restapi/serializer/relationfield.py

```diff
@@ -17,7 +18,12 @@
 @adapter(IRelationValue)
 @implementer(IJsonCompatible)
 def relationvalue_converter(value):
-    if value.to_object:
+    has_permission = api.user.has_permission(
+        "Access Contents Information",
+        obj=value.__parent__
```

Comment:
I do not get why we now check on parent? This is is probably a misunderstanding and wrong.

Reply:
@jensens 

Probably another miss-understanding on my end. 

I used the parent, since @davisagli wrote in the issue #1634 
`@maethu In general, permission to view an object's metadata in listings is controlled by the "Access contents information" permission on its container, not the View permission.`

I'm happy to change this!